sdk/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java

```diff
@@ -291,24 +291,6 @@ Status getStatus() {
     }
   }
 
-  /**
-   * Returns the latency of the {@code Span} in nanos. If still active then returns now() - start
-   * time.
-   *
-   * @return the latency of the {@code Span} in nanos.
-   */
-  long getLatencyNs() {
```

Comment:
I think we need this as a separate field in the SpanData and also available in ReadableSpan to implement what we called z-pages in OpenCensus.

I would suggest to add a new field in SpanData (duration/latency):
* endTime in SpanData is 0 if span is still active.
* duration/latency is now - start (for active) or end - start (for ended).

What do you think?

Reply:
Sounds very reasonable to me.

Reply:
This is implemented now.